src/style.css

```diff
@@ -1 +1,52 @@
-/* add styles here */
+html {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+/* .container {
+  width: 96px;
+
+  height: 96px;
+  display: flex;
+  background-color: #fff;
+  flex-direction: column;
+  justify-content: space-between;
+} */
+
+.stars {
+  display: flex;
+  margin: 0;
+  list-style: none;
+  padding: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+ul.stars--1 li.stars__star:nth-child(-n + 1),
+ul.stars--2 li.stars__star:nth-child(-n + 2),
+ul.stars--3 li.stars__star:nth-child(-n + 3),
+ul.stars--4 li.stars__star:nth-child(-n + 4),
+ul.stars--5 li.stars__star:nth-child(-n + 5) {
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
```suggestion
```

Not needed to add same styles one more time
You already did this on lines 36-37